drivers/volume/portworx/portworx.go

```diff
@@ -158,22 +167,28 @@ func (p *portworx) Init(_ interface{}) error {
 	}
 
 	logrus.Infof("Using %v as endpoint for portworx volume driver", endpoint)
-	endpointPort := fmt.Sprintf("http://%v:%v", endpoint, apiPort)
+	tls := os.Getenv("stork-encryption-enabled")
```

Comment:
Make the env variable a const. Also, use upper case for env variable name

Reply:
Also, this isn't a stork config, but a PX config, so use something like `PX_ENABLE_TLS`